src/AccessibilityInsights.CommonUxComponents/Controls/CommonAutomationPeerCreator.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System.Windows.Automation.Peers;
+using System.Windows.Controls;
+
+namespace AccessibilityInsights.CommonUxComponents.Controls
+{
+    class CommonAutomationPeerCreator
+    {
+        public static AutomationPeer CreateIconAutomationPeer(UserControl owner)
```

Comment:
I'm not sure it's necessarily the case that *all* icons everywhere should be considered decorative/non-content; maybe this method name should specifically mention `DecorativeIcon` or `PresentationalIcon` or `NonContentIcon`?

Reply:
I like `NonContentIcon` or maybe even `ControlViewIcon`